apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/FieldValueResolver.kt

```diff
@@ -0,0 +1,7 @@
+package com.apollographql.apollo.api.internal
+
+import com.apollographql.apollo.api.ResponseField
+
+interface FieldValueResolver<R> {
```

Comment:
This and below 2 classes are shared between regular response parsing and cached response parsing. Moving them to `api` in internal package is the easiest solution. Let me know if you have any concerns. The other option would be to create a new module just for there. Like `http-cache-api` one.

Reply:
I think it should be ok, in future it can be used by KN runtime.